setup.py

```diff
@@ -225,9 +225,9 @@ def build_libtorchtrt_cxx11_abi(
     elif JETPACK_VERSION == "5.0":
         cmd.append("--platforms=//toolchains:jetpack_5.0")
         print("Jetpack version: 5.0")
-    elif JETPACK_VERSION == "6.1":
-        cmd.append("--platforms=//toolchains:jetpack_6.1")
-        print("Jetpack version: 6.1")
+    elif JETPACK_VERSION == "6.2":
+        cmd.append("--platforms=//toolchains:jetpack_6.2")
```

Comment:
We might want to bind version 6.1 and 6.2 to like a 6.x bazel toolchain if there arent really any distinguishing dependencies.

Reply:
I changed it because 6.1 is the same as 6.2
6.2 is the “super nano” update